t/mojo/daemon.t

```diff
@@ -109,6 +109,15 @@ subtest 'Transaction' => sub {
 my $app = Mojolicious->new;
 my $ua  = Mojo::UserAgent->new(ioloop => Mojo::IOLoop->singleton);
 
+my $whatever = 'Failed!';
+$app->hook(
+  before_app_start => sub {
+    my ($app) = shift;
```

Comment:
We don't use shift in list context.

Reply:
Right.  I meant `($app) = @_;` but as your earlier comment indicates, you use `$app = shift` when there is just one argument expected.